extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -373,6 +377,18 @@ export class AmpStoryEmbeddedComponent {
     }
   }
 
+  /**
+   * Schedules embeds to be paused.
+   * @param {!Element} embedEl
+   * @private
+   */
+  scheduleEmbedToPause_(embedEl) {
+    this.resources_.scheduleResume(this.storyEl_, embedEl);
+    if (!this.embedsToBePaused_.includes(embedEl)) {
```

Comment:
I can never remember: are we allowed to use `includes` or are we supposed to use the `find` helpers in `array.js`?

Reply:
Both `includes` (array) and `contains` (string) seem to be used in the runtime :)

Reply:
Yep, it's `find` which we are not supposed to use. I can never remember either >.<